server/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python
+
+WORKDIR /code
+
+COPY ./requirements.txt /code/requirements.txt
+
+RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+
+COPY . /code
+
+EXPOSE 8000
+
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
You can get rid of Dockerfile for server, since docker-compose is a wrapper for managing docker containers, which contains same info as here.